tests/NCronJob.Tests/Helpers/TestHelper.cs

```diff
@@ -192,20 +244,22 @@ private async IAsyncEnumerable<object> GetCompletionJobsAsync(int expectedJobCou
     }
 }
 
-public sealed class Storage
+public sealed class Storage(TimeProvider timeProvider)
 {
 #if NET9_0_OR_GREATER
     private readonly Lock locker = new();
 #else
     private readonly object locker = new();
 #endif
     public IList<string> Entries { get; private set; } = [];
```

Comment:
Couldn't that be just:
```csharp
public IList<string> Entries => TimedEntries.Values.ToList();
```
?

Reply:
PS: We might then need an OrderedDictionary - or something to have a stable order

Reply:
Oh! I like that.

Reply:
Fixed